### PR TITLE
TX gas price bids

### PR DESCRIPTION
**Pull request by pdyraga**
See https://github.com/keep-network/keep-common/pull/36

For every submitted transaction, we check in the predefined intervals if the transaction has been mined already and if not, we increase the gas price by 20% and try to submit again.

The interval at which we check for transaction status, as well as the maximum gas price, can be configured.

### CODE REVIEW

**Comment by pdyraga** [comment]
Example log:
```
21:46:12.967  INFO keep-contr: submitted transaction submitTicket with id: [0x4220237e179243c0d49f7a0f38eaaa2ec338a6fde8310745a492537d572584a0] and nonce [29] KeepRandomBeaconOperator.go:426
21:46:12.970  INFO keep-contr: submitted transaction submitTicket with id: [0x38c59f054edd06dd0c5183e88dddda1cfb9bafd02b9edd588821f7d410a75c7d] and nonce [30] KeepRandomBeaconOperator.go:426
21:46:12.973  INFO keep-contr: submitted transaction submitTicket with id: [0x04f21440e9490f7cd366b506a8ecb6daad0f297b26cad18a0b0f3520a9ad5c69] and nonce [31] KeepRandomBeaconOperator.go:426
21:46:12.976  INFO keep-contr: submitted transaction submitTicket with id: [0x3bd2e333048d9d8c1732f0c3e0d8772479b42a0e5590f5cf00948262594b4f1b] and nonce [32] KeepRandomBeaconOperator.go:426
21:46:12.979  INFO keep-contr: submitted transaction submitTicket with id: [0xa373180e6ed5b0916cf6be92d69e5db7306c97ef2770eb685c13f20084a8820d] and nonce [33] KeepRandomBeaconOperator.go:426
21:46:12.983  INFO keep-contr: submitted transaction submitTicket with id: [0x2877e2e10bc7dc61c8e6b8876d368a0c93f36fa758f083d88e247a413d37b10e] and nonce [34] KeepRandomBeaconOperator.go:426
21:46:12.986  INFO keep-contr: submitted transaction submitTicket with id: [0x1acbb0db38d2adad33233a0aad07546907939a9f3223afda80b89dcae5b9eab1] and nonce [35] KeepRandomBeaconOperator.go:426
21:46:12.988  INFO keep-contr: submitted transaction submitTicket with id: [0x79b9f4a538cde78d3c2f4b10c3075a85371ca5c9f32184eb6e54a3e65fa9eb19] and nonce [36] KeepRandomBeaconOperator.go:426
21:46:12.991  INFO keep-contr: submitted transaction submitTicket with id: [0xbb72c2e7d9898ae3bc2b91ef94cac64c908693012aefcadb1eeed489bcbd2f87] and nonce [37] KeepRandomBeaconOperator.go:426
21:46:12.993  INFO keep-contr: submitted transaction submitTicket with id: [0xf64892827e121f560155eaded9261975c96fd460e35c4db9c9ce1655fcb3593d] and nonce [38] KeepRandomBeaconOperator.go:426
21:46:12.995  INFO keep-contr: submitted transaction submitTicket with id: [0xd223fdfd9d57938bee07d7106c80f9b28c68229588e52e6a225e8beecdf74861] and nonce [39] KeepRandomBeaconOperator.go:426
21:46:12.997  INFO keep-contr: submitted transaction submitTicket with id: [0xe1fdae67ddc47ed30183e1968d7d073d7642e1d10ba43d6e673e90b900b59ae8] and nonce [40] KeepRandomBeaconOperator.go:426
21:46:12.999  INFO keep-contr: submitted transaction submitTicket with id: [0x579f16da1d3daaf0de1d50ac307ebc4b0c8fd9bf7f157831d7abfcdd01a21f69] and nonce [41] KeepRandomBeaconOperator.go:426
21:46:13.001  INFO keep-contr: submitted transaction submitTicket with id: [0x35c55fbf8c06aab9b6564fda0f6b1640e2c90ed9a3739796a0e002c9c0213f5f] and nonce [42] KeepRandomBeaconOperator.go:426
21:46:13.003  INFO keep-contr: submitted transaction submitTicket with id: [0x377387d119365d0e763be24338ed92a7dce45c75f6c06f8d1bef3b4aaef44f20] and nonce [43] KeepRandomBeaconOperator.go:426
21:46:13.005  INFO keep-contr: submitted transaction submitTicket with id: [0xc12d737f7bd2e3affcd86e8479c48f83318a3f0061a2df67227257e62bc8bc0f] and nonce [44] KeepRandomBeaconOperator.go:426
21:46:13.006  INFO keep-group: ticket submission round [10] will start at block [364] and cover tickets with [1] leading zeros groupselection.go:150
21:46:22.970  INFO keep-ethut: transaction [38c59f…a75c7d] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.970  INFO keep-ethut: resubmitting previous transaction [38c59f…a75c7d] with a higher gas price [735] mine_waiter.go:123
21:46:22.971  INFO keep-ethut: transaction [422023…2584a0] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.971  INFO keep-ethut: resubmitting previous transaction [422023…2584a0] with a higher gas price [735] mine_waiter.go:123
21:46:22.974  INFO keep-ethut: transaction [04f214…ad5c69] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.974  INFO keep-ethut: resubmitting previous transaction [04f214…ad5c69] with a higher gas price [735] mine_waiter.go:123
21:46:22.974  INFO keep-contr: submitted transaction submitTicket with id: [0xc443aadcba3a3f6ce67d6d259e5bd3b5110fffebbb5e2bb0adc3aa6fe8259124] and nonce [29] KeepRandomBeaconOperator.go:452
21:46:22.974  INFO keep-contr: submitted transaction submitTicket with id: [0xcf55a6098206569cb1a1570e05c16e1cba08e8c037a92bb4bdc4b43b71d081e7] and nonce [30] KeepRandomBeaconOperator.go:452
21:46:22.976  INFO keep-contr: submitted transaction submitTicket with id: [0x658f829fd885cfb8222c205610ae685a53dbb88e852f3479f45e2fedaf7da70c] and nonce [31] KeepRandomBeaconOperator.go:452
21:46:22.977  INFO keep-ethut: transaction [3bd2e3…4b4f1b] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.977  INFO keep-ethut: resubmitting previous transaction [3bd2e3…4b4f1b] with a higher gas price [735] mine_waiter.go:123
21:46:22.978  INFO keep-contr: submitted transaction submitTicket with id: [0x7bc165815fc0016650a7a4d490c2a02e34caf0036d10364120d674e87286d59c] and nonce [32] KeepRandomBeaconOperator.go:452
21:46:22.980  INFO keep-ethut: transaction [a37318…a8820d] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.980  INFO keep-ethut: resubmitting previous transaction [a37318…a8820d] with a higher gas price [735] mine_waiter.go:123
21:46:22.983  INFO keep-contr: submitted transaction submitTicket with id: [0x838d90d1746bd50edcb67c1a35c5ce76d21bd7b2a531c3b445128950a2dc9e51] and nonce [33] KeepRandomBeaconOperator.go:452
21:46:22.984  INFO keep-ethut: transaction [2877e2…37b10e] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.984  INFO keep-ethut: resubmitting previous transaction [2877e2…37b10e] with a higher gas price [735] mine_waiter.go:123
21:46:22.986  INFO keep-contr: submitted transaction submitTicket with id: [0x056e1d34f093b9098458a600f79d51cc393892d3cc8fceb63047fe37b6a52306] and nonce [34] KeepRandomBeaconOperator.go:452
21:46:22.987  INFO keep-ethut: transaction [1acbb0…b9eab1] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.987  INFO keep-ethut: resubmitting previous transaction [1acbb0…b9eab1] with a higher gas price [735] mine_waiter.go:123
21:46:22.990  INFO keep-ethut: transaction [79b9f4…a9eb19] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.990  INFO keep-ethut: resubmitting previous transaction [79b9f4…a9eb19] with a higher gas price [735] mine_waiter.go:123
21:46:22.990  INFO keep-contr: submitted transaction submitTicket with id: [0x53cd2070515391201bdbf1eefa922d136c4869b590584f372d86fefe8ce13cad] and nonce [35] KeepRandomBeaconOperator.go:452
21:46:22.991  INFO keep-contr: submitted transaction submitTicket with id: [0x426490c305e054e7a64498ab75d47d5f749c8e0b3e270c0fd4e860330dd6aced] and nonce [36] KeepRandomBeaconOperator.go:452
21:46:22.992  INFO keep-ethut: transaction [bb72c2…bd2f87] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.994  INFO keep-ethut: transaction [f64892…b3593d] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:22.999  INFO keep-ethut: resubmitting previous transaction [bb72c2…bd2f87] with a higher gas price [735] mine_waiter.go:123
21:46:22.999  INFO keep-ethut: resubmitting previous transaction [f64892…b3593d] with a higher gas price [735] mine_waiter.go:123
21:46:22.996  INFO keep-ethut: transaction [d223fd…f74861] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.000  INFO keep-ethut: transaction [579f16…a21f69] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.000  INFO keep-ethut: resubmitting previous transaction [d223fd…f74861] with a higher gas price [735] mine_waiter.go:123
21:46:23.000  INFO keep-ethut: resubmitting previous transaction [579f16…a21f69] with a higher gas price [735] mine_waiter.go:123
21:46:22.998  INFO keep-ethut: transaction [e1fdae…b59ae8] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.000  INFO keep-ethut: resubmitting previous transaction [e1fdae…b59ae8] with a higher gas price [735] mine_waiter.go:123
21:46:23.001  INFO keep-contr: submitted transaction submitTicket with id: [0xad51016a1cdf09206df2cd62b5467b11eebe65fb70aad5f352becfc54e36520f] and nonce [38] KeepRandomBeaconOperator.go:452
21:46:23.001  INFO keep-contr: submitted transaction submitTicket with id: [0xcc19d9194b2c16f7ba1e3f6773271027f32f230b0097e43bf131f37cbebdc4d3] and nonce [37] KeepRandomBeaconOperator.go:452
21:46:23.002  INFO keep-ethut: transaction [35c55f…213f5f] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.002  INFO keep-ethut: resubmitting previous transaction [35c55f…213f5f] with a higher gas price [735] mine_waiter.go:123
21:46:23.003  INFO keep-contr: submitted transaction submitTicket with id: [0xab77f5f31b1d81dbd619ae10d05081dd56fa7f7c8b5aa143a0b1aab2978e5a99] and nonce [39] KeepRandomBeaconOperator.go:452
21:46:23.003  INFO keep-contr: submitted transaction submitTicket with id: [0xf3de121af44602b67a1d50af24e1ef57a893024b88166b3f6abe0fb30696378b] and nonce [41] KeepRandomBeaconOperator.go:452
21:46:23.004  INFO keep-ethut: transaction [377387…f44f20] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.004  INFO keep-ethut: resubmitting previous transaction [377387…f44f20] with a higher gas price [735] mine_waiter.go:123
21:46:23.005  INFO keep-contr: submitted transaction submitTicket with id: [0x31cb03264acca043b4cbd00338e72bc43e4acd05dfa165fbf29c792bc4771e31] and nonce [40] KeepRandomBeaconOperator.go:452
21:46:23.005  INFO keep-contr: submitted transaction submitTicket with id: [0xb73ec477d32908c04a91f1bd2dce8367b12a15119157dff7352e6a12b0fd4db3] and nonce [42] KeepRandomBeaconOperator.go:452
21:46:23.006  INFO keep-contr: submitted transaction submitTicket with id: [0xf87920ce307d01f67a8151a31d123a76d883d3ac4c841c5ee55d38035bd98c09] and nonce [43] KeepRandomBeaconOperator.go:452
21:46:23.006  INFO keep-ethut: transaction [c12d73…c8bc0f] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:23.006  INFO keep-ethut: resubmitting previous transaction [c12d73…c8bc0f] with a higher gas price [735] mine_waiter.go:123
21:46:23.008  INFO keep-contr: submitted transaction submitTicket with id: [0x638751452ab878379bd1c13b148db88ba61b9a848281404fd4552b8a474d6cc2] and nonce [44] KeepRandomBeaconOperator.go:452
21:46:32.975  INFO keep-ethut: transaction [c443aa…259124] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.975  INFO keep-ethut: transaction [cf55a6…d081e7] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.975  INFO keep-ethut: resubmitting previous transaction [c443aa…259124] with a higher gas price [882] mine_waiter.go:123
21:46:32.975  INFO keep-ethut: resubmitting previous transaction [cf55a6…d081e7] with a higher gas price [882] mine_waiter.go:123
21:46:32.977  INFO keep-ethut: transaction [658f82…7da70c] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.977  INFO keep-ethut: resubmitting previous transaction [658f82…7da70c] with a higher gas price [882] mine_waiter.go:123
21:46:32.977  INFO keep-contr: submitted transaction submitTicket with id: [0xfded583224eebf520db7d5c286209e7fbc2d4802f9937e256328ff80bf5323c8] and nonce [30] KeepRandomBeaconOperator.go:452
21:46:32.977  INFO keep-contr: submitted transaction submitTicket with id: [0xa3e90e08ed8f641b3a620c068e7a8f6e9441062d1b06375a151d6bdbfbd31eff] and nonce [29] KeepRandomBeaconOperator.go:452
21:46:32.979  INFO keep-ethut: transaction [7bc165…86d59c] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.979  INFO keep-ethut: resubmitting previous transaction [7bc165…86d59c] with a higher gas price [882] mine_waiter.go:123
21:46:32.980  INFO keep-contr: submitted transaction submitTicket with id: [0x5e17a8214192e6adb42e675debc06b3fa9c831179726d104ca62f9a3930d1c1b] and nonce [31] KeepRandomBeaconOperator.go:452
21:46:32.982  INFO keep-contr: submitted transaction submitTicket with id: [0xfb56fcbe5a90ae5d80940dcbbeedc2733746b3f9ec5c2e47db9547ad7f6b3e2a] and nonce [32] KeepRandomBeaconOperator.go:452
21:46:32.984  INFO keep-ethut: transaction [838d90…dc9e51] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.984  INFO keep-ethut: resubmitting previous transaction [838d90…dc9e51] with a higher gas price [882] mine_waiter.go:123
21:46:32.986  INFO keep-contr: submitted transaction submitTicket with id: [0xd54591e13d23efdf7111d340ecd044577843bac7e42614933301947e985cdadb] and nonce [33] KeepRandomBeaconOperator.go:452
21:46:32.989  INFO keep-ethut: transaction [056e1d…a52306] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:32.989  INFO keep-ethut: resubmitting previous transaction [056e1d…a52306] with a higher gas price [882] mine_waiter.go:123
21:46:32.991  INFO keep-contr: submitted transaction submitTicket with id: [0x59c05e0da5a30c7f4745c456aa8a90573bc443b1187800ec2b08a4bc216f0572] and nonce [34] KeepRandomBeaconOperator.go:452
21:46:33.000  INFO keep-ethut: transaction [426490…d6aced] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.000  INFO keep-ethut: transaction [53cd20…e13cad] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.000  INFO keep-ethut: resubmitting previous transaction [426490…d6aced] with a higher gas price [882] mine_waiter.go:123
21:46:33.000  INFO keep-ethut: resubmitting previous transaction [53cd20…e13cad] with a higher gas price [882] mine_waiter.go:123
21:46:33.001  INFO keep-ethut: transaction [ad5101…36520f] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.001  INFO keep-ethut: resubmitting previous transaction [ad5101…36520f] with a higher gas price [882] mine_waiter.go:123
21:46:33.002  INFO keep-contr: submitted transaction submitTicket with id: [0x5fa661dbc578dd73ec5d563a257f84290ed7d3e1e66505a31d09d7551ff12ca5] and nonce [35] KeepRandomBeaconOperator.go:452
21:46:33.002  INFO keep-contr: submitted transaction submitTicket with id: [0xa26c1c685d7705a61aa976b568e397b51bf05ede830692330dabab92ce6c18e5] and nonce [36] KeepRandomBeaconOperator.go:452
21:46:33.003  INFO keep-ethut: transaction [cc19d9…bdc4d3] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.003  INFO keep-ethut: resubmitting previous transaction [cc19d9…bdc4d3] with a higher gas price [882] mine_waiter.go:123
21:46:33.004  INFO keep-ethut: transaction [f3de12…96378b] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.004  INFO keep-ethut: resubmitting previous transaction [f3de12…96378b] with a higher gas price [882] mine_waiter.go:123
21:46:33.004  INFO keep-ethut: transaction [ab77f5…8e5a99] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.004  INFO keep-ethut: resubmitting previous transaction [ab77f5…8e5a99] with a higher gas price [882] mine_waiter.go:123
21:46:33.004  INFO keep-contr: submitted transaction submitTicket with id: [0x3b7bd6d3718ca7a5cb28455089fabd40b5ade54b2936acac9ae986d0127580c1] and nonce [38] KeepRandomBeaconOperator.go:452
21:46:33.005  INFO keep-contr: submitted transaction submitTicket with id: [0xfd4da9e8fa6992c0695cbf72e2eac04c7d023f2d3f62a536a1cf8ea50c47efd6] and nonce [37] KeepRandomBeaconOperator.go:452
21:46:33.005  INFO keep-ethut: transaction [b73ec4…fd4db3] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.010  INFO keep-ethut: resubmitting previous transaction [b73ec4…fd4db3] with a higher gas price [882] mine_waiter.go:123
21:46:33.005  INFO keep-ethut: transaction [31cb03…771e31] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.005  INFO keep-contr: submitted transaction submitTicket with id: [0xdaef34bbe52ad8cd3ef91b015c87ec3c7a703ed3669a2c65a1a1cc7a87982dae] and nonce [39] KeepRandomBeaconOperator.go:452
21:46:33.011  INFO keep-ethut: resubmitting previous transaction [31cb03…771e31] with a higher gas price [882] mine_waiter.go:123
21:46:33.006  INFO keep-contr: submitted transaction submitTicket with id: [0x8fbca4c2568a5ab39330a90899834d9e511edd14460c3e6a1ee691dddb791744] and nonce [41] KeepRandomBeaconOperator.go:452
21:46:33.007  INFO keep-ethut: transaction [f87920…d98c09] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.011  INFO keep-ethut: resubmitting previous transaction [f87920…d98c09] with a higher gas price [882] mine_waiter.go:123
21:46:33.009  INFO keep-ethut: transaction [638751…4d6cc2] not yet mined: [context deadline exceeded] mine_waiter.go:89
21:46:33.011  INFO keep-ethut: resubmitting previous transaction [638751…4d6cc2] with a higher gas price [882] mine_waiter.go:123
21:46:33.012  INFO keep-contr: submitted transaction submitTicket with id: [0x1110b15cc6e6b54c07363137431053fcfbaf8181613167589d2e60a15761fd38] and nonce [42] KeepRandomBeaconOperator.go:452
21:46:33.013  INFO keep-contr: submitted transaction submitTicket with id: [0x1e992119cff49e41918d5f26cef04b015e8bef9008626201932569e2edbf66a2] and nonce [40] KeepRandomBeaconOperator.go:452
21:46:33.013  INFO keep-contr: submitted transaction submitTicket with id: [0xcf4810f82589863bab0bd1c792191eda384601dc187185da403f498987408c0e] and nonce [44] KeepRandomBeaconOperator.go:452
21:46:33.013  INFO keep-contr: submitted transaction submitTicket with id: [0x101ea494a222f46098f478277bc6726da202697228eb442a5bdb01db6a5e9a6a] and nonce [43] KeepRandomBeaconOperator.go:452
21:46:39.980  INFO keep-ethut: transaction [fded58…5323c8] mined with status [1] at block [359] mine_waiter.go:98
21:46:39.980  INFO keep-ethut: transaction [a3e90e…d31eff] mined with status [1] at block [359] mine_waiter.go:98
21:46:39.981  INFO keep-ethut: transaction [5e17a8…0d1c1b] mined with status [1] at block [359] mine_waiter.go:98
21:46:39.984  INFO keep-ethut: transaction [fb56fc…6b3e2a] mined with status [1] at block [359] mine_waiter.go:98
21:46:39.988  INFO keep-ethut: transaction [d54591…5cdadb] mined with status [1] at block [359] mine_waiter.go:98
21:46:39.992  INFO keep-ethut: transaction [59c05e…6f0572] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.003  INFO keep-ethut: transaction [5fa661…f12ca5] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.004  INFO keep-ethut: transaction [a26c1c…6c18e5] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.006  INFO keep-ethut: transaction [3b7bd6…7580c1] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.012  INFO keep-ethut: transaction [daef34…982dae] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.012  INFO keep-ethut: transaction [fd4da9…47efd6] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.012  INFO keep-ethut: transaction [8fbca4…791744] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.014  INFO keep-ethut: transaction [1110b1…61fd38] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.027  INFO keep-ethut: transaction [cf4810…408c0e] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.027  INFO keep-ethut: transaction [1e9921…bf66a2] mined with status [1] at block [359] mine_waiter.go:98
21:46:40.027  INFO keep-ethut: transaction [101ea4…5e9a6a] mined with status [1] at block [359] mine_waiter.go:98
```

File: pkg/chain/ethereum/connect.go [hunk 1]
```diff
@@ -88,6 +116,19 @@ func connect(config ethereum.Config) (*ethereumChain, error) {
...
+	maxGasPrice := DefaultMaxGasPrice
+	if config.MiningCheckInterval != 0 {
+		checkInterval = time.Duration(config.MiningCheckInterval) * time.Second
+	}
+	if config.MaxGasPrice != 0 {
+		maxGasPrice = new(big.Int).SetUint64(config.MaxGasPrice)
+	}
+
+	logger.Infof("Using [%v] mining check interval", checkInterval)
+	logger.Infof("Using [%v] wei max gas price", maxGasPrice)
```
Comment (nkuba) [inline pkg/chain/ethereum/connect.go:129]:
```suggestion
	logger.Infof("using [%v] mining check interval", checkInterval)
	logger.Infof("using [%v] wei max gas price", maxGasPrice)
```

**Comment by pdyraga** [comment]
> Can we add new paramters to the sample config file?

I had it locally, I just forgot to commit this file. 🤦‍♂️

File: go.mod [hunk 2]
```diff
@@ -15,7 +15,7 @@ require (
 	github.com/ipfs/go-datastore v0.1.1
 	github.com/ipfs/go-log v0.0.1
 	github.com/keep-network/go-libp2p-bootstrap v0.0.0-20200423153828-ed815bc50aec
-	github.com/keep-network/keep-common v0.3.3-0.20200512154128-4cefc7b2f12e
+	github.com/keep-network/keep-common v1.0.1-0.20200515214626-9036c7c35946
```
Comment (nkuba) [inline go.mod:18]: Do we want to set it to the commit where we merged the PR to the master? Current one is also fine with me.

Comment (pdyraga) [reply go.mod:18]: I'd leave it like that for now, we'll have to bump up `keep-common` version for the next release anyway.

**Pull request merged**